src/Tizen.NUI/src/public/InputMethodContext.cs

```diff
@@ -359,11 +359,28 @@ public enum KeyboardType
             HardwareKeyboard
         }
 
+        /// <summary>
+        /// Enumeration for the language mode of the input panel.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public enum InputPanelLanguage
+        {
+            /// <summary>
+            /// IME Language automatically set depending on the system display.
```

Comment:
```suggestion
            /// IME Language is automatically set depending on the system display.
```

Reply:
I modify the description in a patch.
https://github.com/Samsung/TizenFX/pull/1623